README.md

```diff
@@ -14,13 +17,15 @@ mkdir migration
 cd migration
 ```
 
-The migration CLI requires two JSON files:
+The migration CLI requires two files:
 
-- _content.json_: the original content to migrate
-- _servers.json_: the Atoti servers' information
+- the file to migrate: it can be the server content stored in a JSON file, _content.json_ or an Atoti jupyter notebook, _notebook.ipynb_.
```

Comment:
```suggestion
- the file to migrate: it can be the server content stored in a JSON file, _content.json_ or an Atoti Jupyter notebook, _notebook.ipynb_.
```